packages/create-sitecore-jss/src/init-runner.ts

```diff
@@ -25,9 +26,8 @@ export const initRunner = async (initializers: string[], args: BaseArgs) => {
       args.silent || console.log(chalk.cyan(`Initializing '${init}'...`));
       const response = await initializer.init(args);
 
-      appName = response.appName;
+      appNames.add(response.appName);
```

Comment:
Just a bit of details here

```suggestion
      // App names can be multiple if the base template requires to setup additional standalone app (e.g. XM Cloud proxy)
      appNames.add(response.appName);
```